app/modules/edit_content.js

```diff
@@ -8,8 +8,8 @@ export default function editTask() {
       if (e.key === 'Enter') {
         e.preventDefault();
```

Comment:
- [ ] [OPTIONAL] When I click a task and make it done and reload the page it becomes again undone it should stay persisted in local storage so please try to do that

Reply:
Hi @youmari,
That feature if I'm not mistaken is part of the last project requirements as stated [HERE](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md) which stage I am not on yet. I'm Still on [todo Add and Remove](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md).🤔 I will add it in the **"todo interactive list part"**.

Reply:
@DaveZag this one is optional you can discard it if you want but better if you do it